libtsuba/src/ParquetWriter.cpp

```diff
@@ -1,9 +1,14 @@
 #include "tsuba/ParquetWriter.h"
 
+#include <memory>
+
 #include <arrow/array/array_binary.h>
+#include <arrow/array/data.h>
 #include <arrow/chunked_array.h>
+#include <arrow/type.h>
 #include <parquet/properties.h>
```

Comment:
Did you intend to add all these arrow headers?

Reply:
Thanks for the call out. I did not. One of these days I'm going to learn how to turn off this feature. the pluggin is too good to drop though.